### PR TITLE
Remove URLQueryItem from swift-corelibs-foundation in favor of FoundationEssentials.URLQueryItem

### PR DESCRIPTION
**Pull request by iCharlesHu**
`URLQueryItem` is already implemented in `FoundationEssentials`.

### CODE REVIEW

**Comment by iCharlesHu** [comment]
@swift-ci please test

File: Sources/Foundation/URLQueryItem.swift [hunk 1]
```diff
@@ -7,96 +7,15 @@
...
-}
+// URLQueryItem is defined in FoundationEssentials
+@_exported import FoundationEssentials
 
 extension URLQueryItem: _NSBridgeable {
     typealias NSType = NSURLQueryItem
-    internal var _nsObject: NSType { return _queryItem }
-}
-
-extension URLQueryItem: _ObjectiveCBridgeable {
```
Comment (jmschonfeld) [inline Sources/Foundation/URLQueryItem.swift:69]: Looks like this change drops the `_ObjectiveCBridgeable` conformance here, was that intentional? I don't think we want to remove it, we'll likely just have to adjust the implementation to call the right initializers (we can use `_nsObject` and `_swiftObject` for convenience)

Comment (iCharlesHu) [reply Sources/Foundation/URLQueryItem.swift:69]: `_ObjectiveCBridgeable` is already implemented in `FoundationEssentials`

**Comment by iCharlesHu** [comment]
Added `_ObjectiveCBridgeable` conformance back.

**Comment by iCharlesHu** [comment]
@swift-ci please test

**Review by jmschonfeld — APPROVED** [review]

**Pull request merged**